### ISSUE TITLE
The data behind all charts should be possible to download as CSV

### ISSUE THREAD

**Issue by marliekeverweij** [quintel/etmodel#3580]
During a meeting with clients yesterday there was the wish to be able to directly download the data behind all charts. We've talked about this improvement before, but figured that this is quite some work considering all the different kinds of charts we have. Maybe we could do this step by step and start with the easiest most common charts. I guess for these charts it's not to hard as we already have a table view:
- stacked bar
- bezier
- mekko

And I guess that these charts are more complex 😉
- sankeys
- hourly charts

**Comment by ChaelKruip** [comment]
Yes! @noracato @antw could you make time estimates for the various types of charts? I think this would be a very nice feature!

**Comment by antw** [comment]
Something to consider for the refresh perhaps? That project probably has substantial implications for the data model behind the charts and we can keep this request in mind while working on that project.

**Comment by noracato** [comment]
Refinement:
- Do only tables (a lot easier)
- What is the exact wish?
- What can we do now that's no regret?
- Check in with the team
